src/zh/cartoonmad/src/eu/kanade/tachiyomi/extension/zh/cartoonmad/CartoonMad.kt

```diff
@@ -0,0 +1,151 @@
+package eu.kanade.tachiyomi.extension.zh.cartoonmad
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.POST
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import okhttp3.FormBody
+import okhttp3.Interceptor
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.Response
+import okhttp3.ResponseBody.Companion.asResponseBody
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+
+class CartoonMad : ParsedHttpSource() {
+    override val baseUrl = "https://www.cartoonmad.com"
+    override val lang: String
+        get() = "zh"
+    override val name: String
+        get() = "動漫狂"
+    override val supportsLatest: Boolean
+        get() = true
```

Comment:
Don't need getters here.
```suggestion
    override val lang: String = "zh"
    override val name: String = "動漫狂"
    override val supportsLatest: Boolean = true
```